src/script/calling/mlsConference.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {Subconversation} from '@wireapp/api-client/lib/conversation/Subconversation';
+import {QualifiedId} from '@wireapp/api-client/lib/user';
+import {MLSService} from '@wireapp/core/lib/messagingProtocols/mls';
+
+import {SubconversationEpochInfoMember} from './CallingRepository';
+
+export const generateSubconversationMembers = async (
```

Comment:
I believe this one can stay private (no need to export it)

Reply:
correct